test/utils/claimToAdjust/JourneyCheckerClaimToAdjustSpec.scala

```diff
@@ -58,11 +58,15 @@ class JourneyCheckerClaimToAdjustSpec extends TestSupport with MockPaymentOnAcco
 
   val whatYouNeedToKnowView: WhatYouNeedToKnow = app.injector.instanceOf[WhatYouNeedToKnow]
 
-  def successfulFutureOk: PoAAmendmentData => Future[Result] = _ =>
-    Future.successful(Ok(whatYouNeedToKnowView(isAgent = false, TaxYear(2023, 2024), showIncreaseAfterPaymentContent = false, "")))
+  def successfulFutureOk: PoAAmendmentData => Future[Result] = _ => {
+    val viewModel = WhatYouNeedToKnowViewModel(poaTaxYear = TaxYear(2023, 2024), showIncreaseAfterPaymentContent = false, "")
```

Comment:
`val viewModel = WhatYouNeedToKnowViewModel(poaTaxYear = TaxYear(2023, 2024), showIncreaseAfterPaymentContent = false, "")`

is used quite a few times. It would be good to move this to the testConstants file
`testConstants/claimToAdjustPOA/ClaimToAdjustPOATestConstants.scala`
and use this val in all places for code maintainability

It would be nice to have an actual redirect location in here too rather than an empty string